subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/ReportDestination.kt

```diff
@@ -0,0 +1,19 @@
+package com.avito.android.test.report.transport
+
+sealed class ReportDestination {
+
+    class Backend(
+        val reportApiUrl: String,
+        val reportViewerUrl: String,
+        val deviceName: String
+    ) : ReportDestination()
+
+    object File : ReportDestination()
+
+    /**
+     * Combination of [File] and [Backend], report.json saved to file, but screenshots and other data uploaded
```

Comment:
This comment signals that Legacy is something unclear
ReportDestination was created on;y for a report without file storage details. 
We should refactor this class and explicitly choose what will be with files in each strategy